servicenow/README.md

```diff
@@ -0,0 +1,152 @@
+# **ServiceNow Integration**
+
+
+## 
+**Setup**
+
+
+### 
+**Prerequisites**
+
+
+
+*   Create a [ServiceNow Developer account](https://developer.servicenow.com/dev.do).
+
+### 
+**Configuring ServiceNow**
+
+*   Log into your ServiceNow Developer account.  If this is the first time you are logging in, you will need to create an _Instance_.  Follow the instructions on creating an Instance and select a _Release_.  As of this writing, Paris is the latest Release and it is recommended that you choose one of the latest releases, Orlando or Paris for this integration.
+*   Click on your Instance URL (ex. _https://dev10101.service-now.com/_) and log in.  You will now be logged into the admin console.
+*   From the Navigation menu, search for “Scripted REST APIs” and select this option.
+*   From the Scripted REST APIs menu, click on the “New” button.
+*   Provide a descriptive name like “Dialogflow Post”.  The API ID field will be automatically populated after you provide a name.  Click on the “Submit” button.
+
+
+
+
+
+![alt_text](images/snow-rest-api.png "image_tooltip")
```

Comment:
Should this be ".images/snow-rest-api.png", as I was not able to see the images through README

Reply:
when I view the readme, I can see the image.